### DISCUSSION TITLE
Why is vcpkg recommended as a git submodule?

### DISCUSSION THREAD

**Discussion by jpr42** [microsoft/vcpkg#27155] [Q&A] [answered]
Currently the [README.md](https://github.com/microsoft/vcpkg/tree/2022.09.27#quick-start-windows) for vcpkg says

"First, download and bootstrap vcpkg itself; it can be installed anywhere, but generally we recommend using vcpkg as a submodule for CMake projects, and installing it globally for Visual Studio projects."

Why is the recommendation submodules for CMake? But global for Visual Studio?

**Comment by Neumann-A** [comment, marked as answer]
My answer is still the same: https://stackoverflow.com/questions/73967245/why-is-vcpkg-recommended-as-a-git-submodule
